utility/src/main/java/org/oppia/android/util/parser/GlideImageLoader.kt

```diff
@@ -51,13 +58,19 @@ class GlideImageLoader @Inject constructor(
       }
     } else imageUrl
 
+    val glideTransformations: Array<Transformation<Bitmap>> = transformations.map {
+      when (it) {
+        ImageTransformation.BLUR -> BlurTransformation(context)
+      }
+    }.toTypedArray()
```

Comment:
I suggest moving this to a helper to avoid the duplication here & above.

Reply:
Done.